apache/collabosphere.template

```diff
@@ -14,12 +14,22 @@
     CustomLog <%= logDirectory %>/collabosphere_access_log common
     CustomLog <%= logDirectory %>/collabosphere-combined_log combined
 
+    RewriteEngine on
+    RewriteCond %{QUERY_STRING} transport=polling
+    RewriteRule /(.*)$ http://localhost:2000/$1 [P]
```

Comment:
Can you add some documentation that this rewrite rule is forcing the request to be proxied to the node app?
Additionally, can you add why we can't add another ProxyPass entry?